plugins/field-colour/package.json

```diff
@@ -43,7 +43,7 @@
     "@blockly/dev-scripts": "^3.1.1",
     "@blockly/dev-tools": "^7.1.6",
     "@typescript-eslint/parser": "^5.59.5",
-    "blockly": "^11.0.0-beta.3",
+    "blockly": "^10.4.3",
```

Comment:
You also need to update the `peerDependencies` for both of these

Reply:
Done.